blocks/facts/__article/facts__article.css

```diff
@@ -0,0 +1,45 @@
+.facts__article {
+    display: flex;
+    width: 100%;
+    margin-top: 25px;
+    margin-bottom: 16px;
+    text-decoration: none;
+}
+
+.facts__article:nth-of-type(2) {
```

Comment:
Usually, it's a bad practice to use `nth` selector

Reply:
agreed, changed it to use last type of instead